### DISCUSSION TITLE
遵守一下原作开源协议

### DISCUSSION THREAD

**Discussion by Gkxie** [Moonofweisheng/wot-design-uni#342] [General]
老哥你这组件库源码搬运的[Wot-Design](https://github.com/jd-ftf/wot-design-mini/commit/d6364a80a4d0230bb94fb036459e185e1009b3ef#diff-686fbdec1651238277e68d188ef78ee9bdd8f8ad60ba557de45d6b14155fa2d7)吧？我看你这组件源码中的注释都和我之前写的一字不差，建议遵守下原作开源协议。

**Comment by Moonofweisheng** [comment]
> 老哥你这组件库源码搬运的[Wot-Design](https://github.com/jd-ftf/wot-design-mini/commit/d6364a80a4d0230bb94fb036459e185e1009b3ef#diff-686fbdec1651238277e68d188ef78ee9bdd8f8ad60ba557de45d6b14155fa2d7)吧？我看你这组件源码中的注释都和我之前写的一字不差，建议遵守下原作开源协议。

确实是参照[Wot-Design](https://github.com/jd-ftf/wot-design-mini/commit/d6364a80a4d0230bb94fb036459e185e1009b3ef#diff-686fbdec1651238277e68d188ef78ee9bdd8f8ad60ba557de45d6b14155fa2d7)做的，基于vue3+TS针对uni-app做了调整，有在首页标注来源，也是MIT协议，请问还有什么需要标注的吗，我这边可以调整。

**Reply by Gkxie** [reply]
可以随意修改项目，但是要保留原作完整License文件

**Reply by Moonofweisheng** [reply]
> 可以随意修改项目，但是要保留原作完整License文件

按我的理解本项目仍是MIT协议，仓库README中包含了指向了wot-design的链接，应该是达到了MIT协议的要求。是否需要将wot-design的官网链接替换为license地址链接呢？下图是相关链接。
![image](https://github.com/Moonofweisheng/wot-design-uni/assets/26426873/6830429e-2ec8-49f1-8934-9852c11cf48a)
